CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+0.2.1 (Feb 1, 2021) -- Bugfix
+
+- Fixes
+
+  - A bug that allowed to clone an OSF project from misshaped URLs of type
+    osf://<project id>/some/where/underneath (was treated as osf://<project
+    id>).
+    This turned into a serious issue, if a dataset had subdatasets and could
+    lead to an endless recursion when being installed via datalad.
```

Comment:
Personally, I would just state the facts and leave the attribution of disaster magnitude to the reader:

> This led to an infinite recursion when installing subdatasets.